waves/package.json

```diff
@@ -37,6 +37,9 @@
     "wasm-pack": "^0.9.1",
     "web-vitals": "^0.2.4"
   },
+  "devDependencies": {
+    "cypress": "^6.2.1"
+  },
```

Comment:
We are using a bundler, hence the distinction between dependencies and dev-dependencies doesn't really do anything. This can go into `dependencies` to avoid the confusion.